pennylane/transforms/hamiltonian_expand.py

```diff
@@ -223,6 +228,12 @@ def hamiltonian_expand(tape: QuantumTape, group: bool = True) -> (Sequence[Quant
     2
     """
 
+    warnings.warn(
+        "'qml.transforms.hamiltonian_expand' is deprecated and will be removed in version 0.39. "
```

Comment:
```suggestion
        "qml.transforms.hamiltonian_expand is deprecated and will be removed in version 0.39. "
```
We don't need (and don't normally do (?)) quotes/backticks in the code-based warnings.

Reply:
Same change for the line below this, and for `sum_expand` as well.